README.md

```diff
@@ -2,92 +2,133 @@
 
-> Additional reading on why stateless is so important to the Eth1/Eth2 merge: https://dankradfeist.de/ethereum/2021/02/14/why-stateless.html)
 
-What is easy to overlook is that such a "stateless" client will be unable to do much else without additional infrastructure.  Specifically it would be unable to serve the vast majority of the JSON-RPC apis.  The Portal Network provides this additional infrastructure, allowing stateless clients to also expose the external APIs that support the web3 ecosystem.
+## Architecture
 
+The Portal Network is built upon on the [Discover V5 protocol](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) and operates over the UDP transport.
 
-### Scalable Lightweight Clients
+The Discovery v5 protocol allows building custom sub-protocols via the use of the built in TALKREQ and TALKRESP message.
 
-The term "light client" tends to refer to a client of the existing DevP2P LES network.  This network is designed using a client/server architecture.  The LES network has a total capacity dictated by the number of "servers" on the network.  In order for this network to scale, the "server" capacity has to increase.  This also means that at any point in time the network has some total capacity which if exceeded will cause service degredation across the network.  Because of this the LES network is unreliable when operating near capacity.
+The Portal Network is divided into the following sub-protocols.
 
-The Portal Network aims to solve this problem by designing our networks so that each additional client that joints the network adds additional capacity to the network.  The end result *should* be a network which becomes more robust and powerful as more nodes join the network.
+- State Network
+- History Network
+- Transaction Gossip Network
+- Header Gossip Network
+- Canonical Indices Network
 
-> Additional reading: https://snakecharmers.ethereum.org/the-winding-road-to-functional-light-clients/
-> 
-> Additional watching: https://www.youtube.com/watch?v=MZxqRs_tLNs
+Each of these networks is designed to deliver a specific unit of functionality.  Most portal clients will participate in all of these networks in order to deliver the full JSON-RPC API.  Each network however is designed to be independent of the others, allowing clients the option of only participating in a subset of them if they wish.
 
+All of the sub-protocols in the Portal Network establish their own Kademlia DHT network that is managed independent of the base Discovery V5 protocol Kademlia DHT.
```

Comment:
While these networks are certainly heavily inspired by Kademlia they're different enough that I don't think they should be called Kademlia.

maybe: "Each of these sub-protocols establish their own DHTs which are managed independently of the discv5 DHT."